doc/Overview/Logging/LoggingOverview.md

```diff
@@ -1,108 +1,174 @@
+
+An option to use Serilog is available by calling the `UseSerilog()` extension method on the `IHostBuilder` instance. This method will configure the Serilog logger to specifically not use the console or file sink by default. The following example shows how to enable Serilog:
 
-public App()
+```csharp
+protected override void OnLaunched(LaunchActivatedEventArgs e)
 {
-    Host = UnoHost
-        .CreateDefaultBuilder()
-        .UseLogging()
-        .UseSerilog()
-        .Build();
-    // ........ //
-}
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => {
+            host
+            .UseLogging()
+            .UseSerilog();
+        });
+...
 ```
 
-## Logging
+For more information about Serilog, check out [Getting started with Serilog](https://github.com/serilog/serilog/wiki/Getting-Started).
 
-To log, you simply need to get a `ILogger` and use the appropriate methods.
+## Uno Internal Logging
+
+To use the same logging system for Uno internal messages call `UseLogging()` on the `IHost` instance:
 
 ```csharp
-var myLogger = myServiceProvider.GetService<ILogger<MyType>>();
+private IHost Host { get; }
 
-myLogger.LogInformation("This is an information log.");
+protected override void OnLaunched(LaunchActivatedEventArgs e)
+{
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => {
+            host
+            .UseLogging()
+        });
+        
+    Host = appBuilder.Build();
+...
 ```
 
-Alternatively you can add `ILogger` as a constructor parameter for any type that's being created by the dependency injection container.
-
+### Setting the XAML Log Level
 
-## Uno Internal Logging
+Sometimes it's necessary to filter messages recorded for specific XAML types to reduce noise. When a preference is specified for the **XAML log level**, the logger will change the verbosity of events from a set of XAML-related types.
 
-To use the same logging for Uno internal messages call EnableUnoLogging after the Build method on the IHost instance. 
+It can be set by calling the `XamlLogLevel()` extension method on the `ILoggingBuilder` instance. The following example shows how to set the XAML log level to `Information`:
 
 ```csharp
-private IHost Host { get; }
+protected override void OnLaunched(LaunchActivatedEventArgs e)
+{
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => {
+            host.UseLogging(
+                builder => {
+                    builder.XamlLogLevel(LogLevel.Information);
+                });
+        });
+...
+```
+
+### Setting the Layout Log Level
 
-public App()
+Similar to the _XAML Log Level_ described above, the **layout log level** can be used to filter messages recorded for specific types to reduce noise. When a preference is specified for the layout log level, the logger will change the verbosity of events from a set of layout-related types.
```

Comment:
I don't think we need dedicated sections to each of the extension methods. Perhaps a table listing the extension method and the assoicated namespaces it sets the filter for